news/2020-03-20-monthly-beatmapping-contest-2-results.md

```diff
@@ -1,40 +1,38 @@
 
 This entry played it relatively safe — it features cleanly organized patterns, intricate slider shapes, and silky smooth gameplay. While you may not earn any performance points from it, it's still a comfortable ride that voters thoroughly enjoyed!
 
 As a reward, Aphestra has earned Monthly Beatmapping Contest's first tier profile badge and two months of osu!supporter.
 
 ![](/wiki/shared/news/2020-02-20-monthly-beatmapping-contest-1-results/mbc-badge-1.png)
 
-**Community vote runner-up:** **[Voicians - Wolves](https://assets.ppy.sh/contests/89/entries/Voicians%20-%20Wolves%20(Reckless%20Beaver).osz?2020)** by **[UndeadCapulet](https://osu.ppy.sh/users/2523533)**, masked as Reckless Beaver.
+---
 
-Garnering **37** votes, UndeadCapulet's entry barely surpassed the third place entry by 1 vote to rise into second place!
+Garnering **37** votes, [UndeadCapulet](https://osu.ppy.sh/users/2523533)'s entry for [Voicians - Wolves](https://assets.ppy.sh/contests/89/entries/Voicians%20-%20Wolves%20(Reckless%20Beaver).osz?2020) barely surpassed the third place entry by 1 vote to rise as the **community vote's runner-up!**
 
 One of the screening crew members, [Uberzolik](https://osu.ppy.sh/users/1314547), noted that while most entries focused on creating low star rating yet engaging maps through slider velocity changes, this one took a different approach. Instead, this entry utilized its small circle size to draw out less conventional cursor movements while also employing unusual rhythms to create a stand-out experience. Give it a play if that's what you're into!
 
 As the contest's runner-up, UndeadCapulet earned a month of supporter and is halfway to the first reward tier's profile badge prize! See the rewards breakdown on the [MBC wiki article's rewards section](https://osu.ppy.sh/help/wiki/Contests/Monthly_Beatmapping_Contest#rewards) for more info about how this works.
 
-**Screening crew's favorite:** **[Voicians - Wolves](https://osu.ppy.sh/beatmapsets/1129015#osu/2358657)** by **[Mao](https://osu.ppy.sh/users/2204515)**, masked as Conscientious Elephant.
+---
 
-Submissions to the MBC are screened by about 10 experienced beatmappers to decide the finalists. This month, Mao's entry placed in the top 5 for 2/3 of those mappers, far surpassing any other entry in terms of screening votes!
+Submissions to this month's MBC were screened by 14 experienced beatmappers to decide the finalists. This month, [Mao](https://osu.ppy.sh/users/2204515)'s entry for (yet again) [Voicians - Wolves](https://osu.ppy.sh/beatmapsets/1129015#osu/2358657) placed in the top 5 for 9 of those mappers, far surpassing any other entry in terms of screening votes and earnings its spot as the **screening crew's favorite!**
```

Comment:
this link is to beatmap page but the others are a download from contests page, idk if it's really a problem that they differ i just wouldn't expect

Reply:
id prioritize osuweb links over contest links, others havent uploaded though